src/Sys/InspectSeoHandler.php

```diff
@@ -0,0 +1,176 @@
+<?php
+declare(strict_types=1);
+
+namespace Triniti\Sys;
+
+use Defuse\Crypto\Crypto;
+use Gdbots\Ncr\Exception\NodeNotFound;
+use Gdbots\Ncr\Ncr;
+use Gdbots\Pbj\Message;
+use Gdbots\Pbjx\CommandHandler;
+use Gdbots\Pbjx\Pbjx;
+use Gdbots\Schemas\Pbjx\StreamId;
+use Gdbots\UriTemplate\UriTemplateService;
+use Google\Service\SearchConsole\InspectUrlIndexResponse;
+use Psr\Log\LoggerInterface;
+use Triniti\Schemas\Sys\Event\SeoInspectedV1;
+use Defuse\Crypto\Key;
+
+
+final class InspectSeoHandler implements CommandHandler
+{
+    public static function handlesCuries(): array
+    {
+        return [
+            'triniti:sys:command:inspect-seo',
+        ];
+    }
+
+    public function __construct(
+        private readonly Ncr        $ncr,
+        private readonly Key        $key,
+        private readonly Flags      $flags,
+        private readonly LoggerInterface $logger,
+    )
+    {
+    }
+
+    public function handleCommand(Message $command, Pbjx $pbjx): void
+    {
+        if ($this->flags->getBoolean('inspect_seo_handler_disabled')) {
+            return;
+        }
+
+        $searchEngines = $command->get('search_engines', ['google']);
+        $nodeRef = $command->get('node_ref');
+
+        try {
+            $node = $this->ncr->getNode($nodeRef);
```

Comment:
@efjacobson @digitalartcrew all pbjx and ncr methods need to pass a context.  this is needed for multi-tenant resolutions to aws resources.  e.g. https://github.com/triniti/core-php/blob/main/src/News/AppleNewsWatcher.php#L136